commons/passive/google/google_sleep_segment_event.avsc

```diff
@@ -0,0 +1,19 @@
+{
+  "namespace": "org.radarcns.passive.google",
+  "type": "record",
+  "name": "GoogleSleepSegmentEvent",
+  "doc": "Represents the result of sleep data after the user is awake.",
+  "fields": [
+    { "name": "time", "type": "double", "doc": "Device timestamp in UTC (s)." },
+    { "name": "timeReceived", "type": "double", "doc": "Device receiver timestamp in UTC (s)." },
+    { "name": "sleepStartTime", "type": "double", "doc": "The UNIX epoch time (s) for the moment when the user goes to sleep." },
+    { "name": "sleepEndTime", "type": "double", "doc": "The UNIX epoch time (s) for the moment when the user wakes up." },
+    { "name": "sleepDuration", "type": "double", "doc": "The amount of elapsed time (s), that the user was asleep." },
```

Comment:
Is the duration the same as `endTime - time`? If so, you can remove the sleepDuration variable. If it is different, please put in the doc what the difference is and rename to `durationAsleep`.

Reply:
OK if it is the same, please remove.